readthedocs/search/faceted_search.py

```diff
@@ -286,6 +288,28 @@ def total_count(self):
         s = s.execute()
         return s.hits.total['value']
 
+    def _get_projects_query(self):
+        """
+        Get filter by projects query.
+
+        If it's a dict, filter by project and version,
+        if it's a list filter by project.
+        """
+        if not self.projects:
+            return None
+
+        if isinstance(self.projects, dict):
+            versions_query = [
+                Bool(filter=[Term(project=project), Term(version=version)])
```

Comment:
Changed from must to filter since we don't want this to add to the score https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html